docs/berkeley-upgrade/archive-migration/migrating-archive-database-to-berkeley.mdx

```diff
@@ -272,16 +272,16 @@ where:
 
 ### Stage 2: Incremental migration
 
-After the initial migration, the data is migrated data up to the last canonical block. However, Devnet/Mainnet data is progressing with new blocks that must also be migrated again and again until the fork block is announced. 
+After the initial migration, the data is migrated data up to the last canonical block. However, Devnet/Mainnet data is progressing with new blocks that must also be migrated again and again until the fork block is announced.
 
 :info: Incremental migration can, and probably must, be repeated a couple of times until the fork block is announced by Mina Foundation.
```

Comment:
info box not working ... need :::info closed by :::

Reply:
This should be sorted out now.

Reply:
Also see **info** on line 424 - should this also be a "box" by using :::info ::: ?